2018-11-14 18:38:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Value too long for column "CVV VARCHAR(3)": "'51928' (5)"; SQL statement:
INSERT INTO PAYMENT_INFO (CONFIRMATION_NUMBER, CARD_NUM, CVV, EXP_DATE, HOLDER_NAME) VALUES (?, ?, ?, ?, ?) [22001-197]
2018-11-19 18:38:54 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Chel/source/cse5234/MyRestaurantDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at edu.osu.cse5234.batch.InventoryUpdater.createConnection(InventoryUpdater.java:31)
	at edu.osu.cse5234.batch.InventoryUpdater.main(InventoryUpdater.java:18)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Chel/source/cse5234/MyRestaurantDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 26 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/Chel/source/cse5234/MyRestaurantDB.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 20 more
2018-11-19 18:44:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "UPDATE ITEM SET AVAILABLE_QUANTITY = 6 FROM[*] ITEM WHERE ITEM_NUMBER = 1 "; SQL statement:
update ITEM set AVAILABLE_QUANTITY = 6 from ITEM where ITEM_NUMBER = 1 [42000-197]
2018-11-19 19:17:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-197]
2018-11-26 15:35:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-197]
2018-11-26 15:38:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-197]
